test/docker/test_readiness.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+check_readiness() {
```

Comment:
Suggestion, write the script in ruby or node.js?

Reply:
I could! I was just trying to keep it as simple of a script as possible